parts/linux/cloud-init/artifacts/components.json

```diff
@@ -224,7 +224,8 @@
         "v1.29.4",
         "v1.29.9",
         "v1.30.0",
-        "v1.30.5"
+        "v1.30.5",
+        "v1.31.0"
```

Comment:
can we start thinking about removing older versions?

VHD space is filling up quickly

Reply:
Will remove once 1.27 is out